.github/workflows/release.yml

```diff
@@ -66,14 +68,23 @@ jobs:
           rm -rf src
       - name: Test generated standalone binary
         run: yarn dist-standalone:test
-      - name: Upload binaries to GitHub release
-        uses: shogo82148/actions-upload-release-asset@v1
```

Comment:
Why change from this upload-release-asset action?

Reply:
Because we don't need it: it's unnecessary external code for doing something that we can easily do with the API through the `github-script@v7` action, just like our other API calls